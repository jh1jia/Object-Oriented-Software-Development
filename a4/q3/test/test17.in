test16.txt dropfirst 0
