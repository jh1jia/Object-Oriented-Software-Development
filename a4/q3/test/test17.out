1111111111
2222222222
3333333333
0123456789
9876543210
7861092364
1
2
3
4
5
6
7
0
9
